### PR TITLE
Data Visualization: Update palette and add new error tokens

### PR DESCRIPTION
**Pull request by ayeshakmaz**
#### What changed?

Update values used for data visualization palette
Deprecate $color-data-visualization-error
Add $color-data-visualization-error-graph and $color-data-visualization-error-text

Screenshot 2023-01-17 at 9 47 22 AM

Before
Screen Shot 2023-01-17 at 5 57 33 PM

After

Screen Shot 2023-01-17 at 5 57 40 PM

#### Why?

Create better accessibility and stronger contrast 
Hard coded colors are improved contrast/brand - may stay hard coded for now until brand updates to base palette

### Links

- [Jira](https://jira.pinadmin.com/browse/GESTALT-XXXX)

- [Figma](https://www.figma.com/file/0Bh7hWYgnwhmUTLYnfayUS/branch/nbOXLJsFt62qQS8s7nSZnK/Library%3A-Data-viz-colors?node-id=1001%3A9176&t=mlrxHj7pQHJF3tIs-0)

### Checklist

- [ ] Added unit and Flow Tests
- [ ] Added documentation + accessibility tests
- [ ] Verified accessibility: keyboard & screen reader interaction
- [ ] Checked dark mode, responsiveness, and right-to-left support
- [ ] Checked stakeholder feedback (e.g. Gestalt designers)

### CODE REVIEW

**Comment by AlbertCarreras** [comment]
@ayesha can we add before and after of the token changes?

**Comment by AlbertCarreras** [comment]
@ayeshakmaz Can we add a little bit more context on why we are moving from token values to custom values? I assume is contract etc But for context, are we planning on add tokens for these as well? I feel some more context could be good.


Screenshot 2023-01-17 at 9 52 08 AM

File: docs/graphics/data-viz-color-examples/dataVizSuccessError.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...on-error-text)" d="M493.882 106.13h1.914c0-.964.629-1.641 1.634-1.641.964 0 1.524.643 1.524 1.449 0 .684-.266 1.135-1.497 2.332l-3.459 3.342V113h7.116v-1.661h-4.334v-.123l2.051-1.955c1.641-1.559 2.147-2.352 2.147-3.507 0-1.661-1.34-2.871-3.432-2.871-2.16 0-3.664 1.326-3.664 3.247Zm10.848 1.606c1.477 0 2.345-.882 2.345-2.399 0-1.483-.888-2.393-2.345-2.393-1.449 0-2.344.916-2.344 2.393 0 1.511.868 2.399 2.344 2.399Zm1.012 5.264 6.556-9.864h-1.579L504.163 113h1.579Zm-1.012-6.453c-.499 0-.786-.438-.786-1.203 0-.745.294-1.203.786-1.203.486 0 .78.458.78 1.203 0 .759-.281 1.203-.78 1.203Zm7 6.562c1.477 0 2.345-.881 2.345-2.399 0-1.483-.888-2.393-2.345-2.393-1.449 0-2.344.916-2.344 2.393 0 1.511.875 2.399 2.344 2.399Zm0-1.189c-.499 0-.779-.438-.779-1.203 0-.745.287-1.203.779-1.203.493 0 .787.458.787 1.203 0 .759-.288 1.203-.787 1.203Z"/><path stroke="var(--color-data-visualization-error-graph)" stroke-linecap="round" stroke-linejoin="round" stroke-width="3" d="m418 177.203 9.435-4.603 8.845 4.603 10.467-11.762 8.845 17.388 9.73-48.329 10.319 14.575 8.993 41.425 10.024-7.671h8.551l9.582-30.174 10.319 12.786h8.551l9.287 14.319L558 173.93"/><path stroke="#E9E9E9" d="M.5.5h735v261H.5z"/></svg>
```
Comment (AlbertCarreras) [inline docs/graphics/data-viz-color-examples/dataVizSuccessError.svg:1]: @ayeshakmaz I dont see anything here?

**Review by AlbertCarreras — APPROVED** [review]
LGTM, left some comments

File: docs/graphics/data-viz-color-examples/dataVizSuccessError.svg (hunk 1 above)
Comment (ayeshakmaz) [reply docs/graphics/data-viz-color-examples/dataVizSuccessError.svg:1]: It's changes to some of the tokens used in the code (text color mainly)

File: docs/pages/foundations/data_visualization/usage.js [hunk 2]
```diff
@@ -1,17 +1,45 @@
...
-import DataViz8Colors from '../../../graphics/color-examples/dataViz8Colors.svg';
+import DataVizPrimary from '../../../graphics/data-viz-color-examples/dataVizPrimary.svg';
+import DataVizSuccessError from '../../../graphics/data-viz-color-examples/dataVizSuccessError.svg';
+import DataViz2Colors from '../../../graphics/data-viz-color-examples/dataViz2Colors.svg';
+import DataViz3Colors from '../../../graphics/data-viz-color-examples/dataViz3Colors.svg';
+import DataViz4Colors from '../../../graphics/data-viz-color-examples/dataViz4Colors.svg';
+import DataViz6Colors from '../../../graphics/data-viz-color-examples/dataViz6Colors.svg';
+import DataViz8Colors from '../../../graphics/data-viz-color-examples/dataViz8Colors.svg';
+
+const COLOR_TEXT_PAIRINGS = {
```
[the remark this answers is not in the document]
Comment (ayeshakmaz) [reply docs/pages/foundations/data_visualization/usage.js:16]: Good catch

**Pull request merged**